helm/templates/configmap.yaml

```diff
@@ -15,13 +15,13 @@ data:
   RESPONSE_COMPRESSION_ENABLED: {{ .Values.env.responseCompressionEnabled | quote }}
   LOG_LEVEL: {{ .Values.env.logLevel | quote }}
   LOG_PRETTY_PRINT_ENABLED: {{ .Values.env.logPrettyPrintEnabled | quote }}
-  {{ if .Values.env.tracing.enabled }}
-  TELEMETRY_TRACING_ENABLED: 'true'
-  TELEMETRY_TRACING_URL: {{ $tracingUrl }}
+  TELEMETRY_TRACING_ENABLED: {{ $tracing.enabled | quote }}
```

Comment:
create an issue in [ts-server-boilerplate.](https://github.com/MapColonies/ts-server-boilerplate) about the 'false' case.